### ISSUE TITLE
Accessibility violation in canvas node label edit icon

### ISSUE THREAD

**Issue by srikant-ch5** [elyra-ai/canvas#2499]
Ref: 4049

<img width="745" alt="Image" src="https://github.com/user-attachments/assets/216681eb-e838-4742-8ac7-3176d2a03b0f" />
